testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/JVMServerPropertiesTestCase.java

```diff
@@ -0,0 +1,209 @@
+        Properties p = new Properties();
+        p.load(new StringReader(response));
+
+        Assert.assertEquals(serverBaseDir.toAbsolutePath().toString(), p.getProperty("test.jboss.server.base.dir"));
+        Assert.assertEquals(serverLogDir.toAbsolutePath().toString(), p.getProperty("test.jboss.server.log.dir"));
+        Assert.assertEquals(serverDataDir.toAbsolutePath().toString(), p.getProperty("test.jboss.server.data.dir"));
+        Assert.assertEquals(serverTmpDir.toAbsolutePath().toString(), p.getProperty("test.jboss.server.temp.dir"));
+    }
+
+    private static String performHttpCall(String host, int port, String context) throws IOException {
+        URLConnection conn;
+        InputStream in = null;
+        StringWriter writer = new StringWriter();
+        try {
+            URL url = new URL("http://" + TestSuiteEnvironment.formatPossibleIpv6Address(host) + ":" + port + "/" + context);
+            conn = url.openConnection();
+            conn.setDoInput(true);
+            in = new BufferedInputStream(conn.getInputStream());
+            int i = in.read();
+            while (i != -1) {
+                writer.write((char) i);
+                i = in.read();
+            }
+            return writer.toString();
+        } finally {
+            safeClose(in);
+            safeClose(writer);
+        }
+    }
+
+
+    private ModelNode createDeploymentOperation(Path deployment, PathElement... serverGroups) throws MalformedURLException {
+        ModelNode content = new ModelNode();
+        content.get("url").set(deployment.toUri().toURL().toString());
+
+        ModelNode op = Util.createEmptyOperation(COMPOSITE, PathAddress.EMPTY_ADDRESS);
+        ModelNode steps = op.get(STEPS);
+        ModelNode step1 = steps.add();
+        step1.set(Util.createEmptyOperation(ADD, PathAddress.pathAddress(PathElement.pathElement(DEPLOYMENT, deployment.getFileName().toString()))));
+        step1.get(CONTENT).add(content);
+        for (PathElement serverGroup : serverGroups) {
+            ModelNode sg = steps.add();
+            sg.set(Util.createEmptyOperation(ADD, PathAddress.pathAddress(serverGroup, PathElement.pathElement(DEPLOYMENT, deployment.getFileName().toString()))));
+            sg.get(ENABLED).set(true);
+        }
+
+        return op;
+    }
+
+    public static WebArchive createDeployment() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, PROP_SERVLET_APP_WAR);
+        war.addClass(PropertiesServlet.class);
+        war.addAsManifestResource(createPermissionsXmlAsset(
+                new PropertyPermission("test.jboss.server.base.dir", "read"),
+                new PropertyPermission("test.jboss.server.log.dir", "read"),
+                new PropertyPermission("test.jboss.server.data.dir", "read"),
+                new PropertyPermission("test.jboss.server.temp.dir", "read")
+        ), "permissions.xml");
+
+        return war;
+    }
+}
```

Comment:
Please add new line to end of file. At least github show that there is no new line (only in this file).

Reply:
done